pkg/images/images.go

```diff
@@ -12,9 +12,10 @@ const (
 	PrometheusServerImageName string = "prometheus:v2.42.0"
 	OauthProxyImageRegistry   string = "quay.io/openshift"
 	OauthProxyImageName       string = "origin-oauth-proxy:4.14.0"
+)
```

Comment:
`NetworkConsoleCollector` is now `NetworkObserver`

So this line:
```
NetworkConsoleCollectorImageName string = "network-console-collector:v2-latest"
```

should be changed to
```
NetworkObserverImageName string = "network-observer:v2-latest"
```

Reply:
done